client/src/components/Header/Header.js

```diff
@@ -1,52 +1,11 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-import Button from "../Button/Button";
 import "./header.css";
 
-const Header = ({ user, onLogin, onLogout, onCreateAccount }) => (
-  <header>
-    <div className="wrapper">
-      <div>
-        <svg
-          width="32"
-          height="32"
-          viewBox="0 0 32 32"
-          xmlns="http://www.w3.org/2000/svg"
-        >
-          <g fill="none" fillRule="evenodd">
-            <path
-              d="M10 0h12a10 10 0 0110 10v12a10 10 0 01-10 10H10A10 10 0 010 22V10A10 10 0 0110 0z"
-              fill="#FFF"
-            />
-            <path
-              d="M5.3 10.6l10.4 6v11.1l-10.4-6v-11zm11.4-6.2l9.7 5.5-9.7 5.6V4.4z"
-              fill="#555AB9"
-            />
-            <path
-              d="M27.2 10.6v11.2l-10.5 6V16.5l10.5-6zM15.7 4.4v11L6 10l9.7-5.5z"
-              fill="#91BAF8"
-            />
-          </g>
-        </svg>
-        <h1>Acme</h1>
-      </div>
-      <div>
-        {user ? (
-          <Button size="small" onClick={onLogout} label="Log out" />
-        ) : (
-          <>
-            <Button size="small" onClick={onLogin} label="Log in" />
-            <Button
-              primary
-              size="small"
-              onClick={onCreateAccount}
-              label="Sign up"
-            />
-          </>
-        )}
-      </div>
-    </div>
+const Header = () => (
+  <header className="wrapper">
```

Comment:
The absolute benefit of styled-components is that you don't need to give the elements any classNames this line you could write as follows: 
```suggestion
  <HeaderWrapper>
   <h1>NEOWISE - PROJECT </h1>
   </HeaderWrapper>
   
   const HeaderWrapper = styled.header`
    font-family: "Orbitron", Arial, sans-serif;
    font-weight: 700;
    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
    padding: 15px 20px;
   display: flex;
   align-items: center;
   
   svg {
     justify-content: center;
   }
   
   h1 { 
    font-weight: 900;
   letter-spacing: 0.8rem;
  font-size: 30px;
  color: #fff;
  line-height: 1;
  margin: 20px 0 20px;
  text-align: center;
   }
    `;
```

Reply:
Yes, I´m also a fan of styled-componts. Just have to clean up work to to..